### ISSUE TITLE
failed to start kernal -  wrong architecture

### ISSUE THREAD

**Issue by fdq09eca** [microsoft/vscode-jupyter#10410]
I want to run the `.ipynb` in a `venv` but it failed.

```
Failed to start the Kernel. 
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.9/3.9.12_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.9/3.9.12_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel_launcher.py", line 15, in <module>
    from ipykernel import kernelapp as app
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel/kernelapp.py", line 51, in <module>
    from .ipkernel import IPythonKernel
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel/ipkernel.py", line 17, in <module>
    from .comm import CommManager
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel/comm/__init__.py", line 1, in <module>
    from .comm import *  # noqa
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel/comm/comm.py", line 12, in <module>
    from ipykernel.kernelbase import Kernel
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/ipykernel/kernelbase.py", line 35, in <module>
    import psutil
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/psutil/__init__.py", line 123, in <module>
    from . import _psosx as _psplatform
  File "/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/psutil/_psosx.py", line 14, in <module>
    from . import _psutil_osx as cext
ImportError: dlopen(/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/psutil/_psutil_osx.cpython-39-darwin.so, 2): no suitable image found.  Did find:
	/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/psutil/_psutil_osx.cpython-39-darwin.so: mach-o, but wrong architecture
	/Users/chrislam/Documents/UoH-GIS-2021/venv/lib/python3.9/site-packages/psutil/_psutil_osx.cpython-39-darwin.so: mach-o, but wrong architecture. 
View Jupyter [log](command:jupyter.viewOutput) for further details.
```

see the following information, if any worth.

`venv python version`

**![image](https://user-images.githubusercontent.com/51902242/173053196-81e4a59c-cc42-485a-b7cc-3ea43ccaec7c.png)**

`ipykernel.__version__`

```py
>>>
Python 3.9.12 (main, May  8 2022, 17:57:54) 
[Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipykernel
>>> ipykernel.__version__
'6.13.1'
```

`vscode version`

```
Version: 1.66.2
Commit: dfd34e8260c270da74b5c2d86d61aee4b6d56977
Date: 2022-04-11T07:49:20.994Z
Electron: 17.2.0
Chromium: 98.0.4758.109
Node.js: 16.13.0
V8: 9.8.177.11-electron.0
OS: Darwin x64 20.6.0
```

`ipynb version`
![image](https://user-images.githubusercontent.com/51902242/173053076-a40deb3c-5e00-4304-8686-b8232df68b73.png)

**Comment by amunger** [comment]
vscode and the jupyter extension are both out of date by a couple of releases, can you try updating to see if that fixes the issue?
vscode just released 1.68 and jupyter is at v2022.5

**Comment by fdq09eca** [comment]
I have updated the latest vscode and Jupyter notebook extension, but this problem still exists.
however, if I am using the following python as a kernel, things are fine, but it is not what I want. I will need the packages in the `venv` to execute the notebook.
![image](https://user-images.githubusercontent.com/51902242/174094328-a1395f75-ab9b-44ea-9958-38aefb07c935.png)
